packages/SwingSet/src/kernel/kernel.js

```diff
@@ -1013,7 +1046,7 @@ export default function buildKernel(
         const source = { bundle };
         const vatID = kernelKeeper.allocateUnusedVatID();
         const event = { type: 'create-vat', vatID, source, dynamicOptions };
-        kernelKeeper.addToRunQueue(harden(event));
+        kernelKeeper.addToAcceptanceQueue(harden(event));
```

Comment:
Hm.. FYI, this call only happens while the `vatAdmin` vat is running, because it's (normally) the only one with access to `devices.vatAdmin`, and this is an endowment provided only to the device. Putting it on the acceptance queue kinda claims to rate-limit the creation of new vats, but that's already limited by the ability to deliver `vatAdminService~.createVat()` deliveries into vatAdmin.

I think it might be better to drop this on the other queue. Or to remember (maybe add a note) that this wants to go onto its own queue (not associated with any particular vat). The `create-vat` events are not vat syscalls (even though they're triggered by a `syscall.callNow(vatAdminDevice)` from vatAdminVat), and `devices.vatAdmin` does not know which vat is asking it to use its kernel endowment, and we're not considering having device-outbound queues to match the vat-outbound queues.

Reply:
> Or to remember (maybe add a note) that this wants to go onto its own queue (not associated with any particular vat).

Planning on revisiting all enqueuing in the next PR. As stated I'm trying not to change any order in this PR.